test/Ocelot.AcceptanceTests/PollyQoSTests.cs

```diff
@@ -38,7 +38,7 @@ public override void Dispose()
     public void Should_not_timeout()
     {
         var port = PortFinder.GetRandomPort();
-        var route = GivenRoute(port, new QoSOptions(10, 500, 1000, null), HttpMethods.Post);
+        var route = GivenRoute(port, new QoSOptions(10, 500, .5, 5, 1000, null), HttpMethods.Post);
```

Comment:
### Test readability is low❗ 
We have to switch to `QoSOptionsBuilder` in tests avoiding direct usage of constructors!
With `QoSOptionsBuilder` test readability will be excellent. 😉